---
title: "' ' は ' <name1> ' から継承されるため、' Catch ' ブロックに到達しませんでした。 <name2>"
ms.date: 07/20/2015
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
ms.openlocfilehash: 560beddd1e9051c43d610fe893a55baa2a5c559a
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 06/04/2020
ms.locfileid: "84395650"
---
# <a name="catch-block-never-reached-because-name1-inherits-from-name2"></a><span data-ttu-id="1444a-102">' ' は ' \<name1> ' から継承されるため、' Catch ' ブロックに到達しませんでした。 \<name2></span><span class="sxs-lookup"><span data-stu-id="1444a-102">'Catch' block never reached, because '\<name1>' inherits from '\<name2>'</span></span>
<span data-ttu-id="1444a-103">コード内の `Catch` ブロックに到達することはありません。</span><span class="sxs-lookup"><span data-stu-id="1444a-103">A `Catch` block within the code is never reached.</span></span>  
  
 <span data-ttu-id="1444a-104">既定では、このメッセージは警告です。</span><span class="sxs-lookup"><span data-stu-id="1444a-104">By default, this message is a warning.</span></span> <span data-ttu-id="1444a-105">警告を非表示にする方法や、警告をエラーとして扱う方法の詳細については、「 [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic)」を参照してください。</span><span class="sxs-lookup"><span data-stu-id="1444a-105">For more information on hiding warnings or treating warnings as errors, see [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span></span>  
  
 <span data-ttu-id="1444a-106">**エラー ID:** BC42029</span><span class="sxs-lookup"><span data-stu-id="1444a-106">**Error ID:** BC42029</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="1444a-107">このエラーを解決するには</span><span class="sxs-lookup"><span data-stu-id="1444a-107">To correct this error</span></span>  
  
- <span data-ttu-id="1444a-108">余分な `Catch` ブロックを削除します。</span><span class="sxs-lookup"><span data-stu-id="1444a-108">Remove the extraneous `Catch` block.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="1444a-109">関連項目</span><span class="sxs-lookup"><span data-stu-id="1444a-109">See also</span></span>

- [<span data-ttu-id="1444a-110">Try/catch ブロックを使用して例外をキャッチする方法</span><span class="sxs-lookup"><span data-stu-id="1444a-110">How to use the try/catch block to catch exceptions</span></span>](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [<span data-ttu-id="1444a-111">Try...Catch...Finally ステートメント</span><span class="sxs-lookup"><span data-stu-id="1444a-111">Try...Catch...Finally Statement</span></span>](../language-reference/statements/try-catch-finally-statement.md)
